docs/standard/io/how-to-convert-between-dotnet-streams-and-winrt-streams.md

```diff
@@ -9,65 +9,58 @@ ms.assetid: 23a763ea-8348-4244-9f8c-a4280b870b47
+To convert from a Windows Runtime random-access stream to a .NET Framework stream, use the [System.IO.WindowsRuntimeStreamExtensions.AsStream](https://msdn.microsoft.com/library/system.io.windowsruntimestreamextensions.asstream.aspx) method.
 
-The Windows Runtime offers stream types that support reading only, writing only or reading and writing, and these capabilities are maintained when you convert a Windows Runtime stream to a .NET Framework stream. Furthermore, if you convert a Windows Runtime stream to a .NET Framework stream and back, you get the original Windows Runtime instance back. It’s best practice to use the conversion method that matches the capabilities of the Windows Runtime stream you would like to convert. However, since <xref:Windows.Storage.Streams.IRandomAccessStream> is readable and writeable (it implements both <xref:Windows.Storage.Streams.IOutputStream> and <xref:Windows.Storage.Streams.IInputStream>, you can use any of the conversion methods and the capabilities of the original stream are maintained. For example, using [System.IO.WindowsRuntimeStreamExtensions.AsStreamForRead](https://msdn.microsoft.com/library/system.io.windowsruntimestreamextensions.asstreamforread.aspx) to convert an <xref:Windows.Storage.Streams.IRandomAccessStream> will not limit the converted .NET Framework stream to being readable only; it will also be writable.
+The following code example prompts you to select a file, opens it with Windows Runtime APIs, and then converts it to a .NET Framework stream. It reads the stream and outputs it to a text block. You would typically manipulate the stream with .NET Framework APIs before outputting the results.
 
-### To convert from a Windows Runtime random-access stream to a .NET Framework stream
-
-- Use the [System.IO.WindowsRuntimeStreamExtensions.AsStream](https://msdn.microsoft.com/library/system.io.windowsruntimestreamextensions.asstream.aspx) method.
-
-  The following code example shows how to prompt the user to select a file, open it with Windows Runtime APIs, and then convert it to a .NET Framework stream, which is read and output to a text block. In this scenario, you would typically manipulate the stream with .NET Framework APIs before outputting the results.
-
-  To run this example, you must create a Windows Store XAML app that contains a text block named `TextBlock1` and a button named  `Button1`. The button click event must be associated with the `button1_Click` method shown in the example.
+To run this example, you must create a UWP XAML app that contains a text block named `TextBlock1` and a button named  `Button1`. The button click event must be associated with the `button1_Click` method shown in the example.
 
   [!code-csharp[System.IO.WindowsRuntimeStreamExtensionsEx#Imports](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.io.windowsruntimestreamextensionsex/cs/mainpage.xaml.cs#imports)]
```

Comment:
The code here is awkward, since it's broken into two blocks, one for imports and one for the event handler. I'd use line highlighting. I'd remove all snippet tags from the file, remove the snippet number references from here and the next code reference, and highlight the line numbers of the event handler. For example, for the C# file (after an unnecessary blank line is also removed), it should be:

[!code-csharp[System.IO.WindowsRuntimeStreamExtensionsEx#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.io.windowsruntimestreamextensionsex/cs/mainpage.xaml.cs#L23-60)]